src/popup/components/Header.vue

```diff
@@ -187,6 +202,20 @@ export default defineComponent({
       ionRouter.navigate({ name: currentHomeRouteName.value }, 'back', 'push');
     }
 
+    watch(
+      () => props.show,
+      async (value) => {
+        if (value) {
+          showHeader.value = true;
+        } else {
+          // wait for the page transition to finish
+          await new Promise((resolve) => setTimeout(resolve, 150));
+          showHeader.value = false;
```

Comment:
Maybe just:
```suggestion
          setTimeout(() => {
            showHeader.value = false;
          }, 150));
```

Also maybe save this `150` magic number to some kind of constant to describe what it actually is.